supabase/templates/welcome.html

```
@@ -0,0 +1,97 @@
+<div style="
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)